README.md

```diff
@@ -1 +1,58 @@
-# go-appr
+## go-appr
+`go-swagger` generated client bindings for `App Registry`. This package can be used to talk to `quay.io` or [appr](https://github.com/app-registry/appr) to pull down application packages.
+
+Swagger spec file has been obtained from here -  [appr-api-swagger.yaml](https://github.com/app-registry/appr/blob/master/Documentation/server/appr-api-swagger.yaml)
+
+### Prerequisite
```

Comment:
It is unclear to me if this is a prerequisite for using go-appr or if you are describing the steps that you followed to generate this code? If it is the latter, then the section should be renamed. Moreover I would suggest this readme to focus on how to use go-appr. We could have another file where you describe how you generated the code.